hearth/lib/hearth/http/fields.rb

```diff
@@ -103,6 +103,10 @@ def each(&block)
                  .each(&block)
         end
         alias each_pair each
+
+        def inspect
+          "#<#{self.class.name}>"
```

Comment:
Why do we do this?

Reply:
Response output is very verbose because Headers/Trailers will show the value of Fields too. It's duplicate information.